deploy/examples/crds.yaml

```diff
@@ -4968,7 +4971,6 @@ spec:
                         type: object
                         x-kubernetes-preserve-unknown-fields: true
                     type: object
-                  nullable: true
```

Comment:
We have had upgrade problems with things being turned from `nullable` to non-nullable. Should we keep this?

Reply:
We really do require at least one data pool, so not sure there would be an impact here either way.